src/lib/crypto/kimchi_bindings/stubs/kimchi_types.ml

```diff
@@ -206,11 +193,9 @@ module VerifierIndex = struct
       ; lookup_info : lookup_info
       ; runtime_tables_selector : 'poly_comm option
       }
-    [@@boxed]
   end
 
   type nonrec 'fr domain = { log_size_of_group : int; group_gen : 'fr }
-  [@@boxed]
 
```

Comment:
Those changes seem like a side effect of running unit tests. I also saw those changes being made after I ran `testone.sh` script.

Reply:
I mean remove the `[@@boxed]`